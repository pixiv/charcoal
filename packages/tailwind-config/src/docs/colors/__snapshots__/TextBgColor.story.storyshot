// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots tailwind-config/Colors/Text bg color Playground 1`] = `
<div
  data-dark={false}
>
  <div
    className="bg-background1 p-64 max-w-2xl"
  >
    <p
      className="typography-20 text-text1"
    >
      charcoal はピクシブ株式会社のデザインシステムです。ここでは特に、Web フロントエンドの実装に用いる npm パッケージ集のことを言います。Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at odio bibendum nisl mollis eleifend et quis turpis. Quisque dignissim porta justo ut convallis.dipiscing elit.
    </p>
  </div>
</div>
`;
