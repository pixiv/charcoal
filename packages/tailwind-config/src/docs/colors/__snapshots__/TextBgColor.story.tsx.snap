// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tailwind-config/Colors/Text bg color Playground smoke-test 1`] = `
<div data-dark="false">
  <div class="bg-background1 p-64 max-w-2xl">
    <p class="typography-20 text-text1">
      charcoal はピクシブ株式会社のデザインシステムです。ここでは特に、Web フロントエンドの実装に用いる npm パッケージ集のことを言います。Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at odio bibendum nisl mollis eleifend et quis turpis. Quisque dignissim porta justo ut convallis.dipiscing elit.
    </p>
  </div>
</div>
`;
